test2.py

```diff
@@ -1,40 +1,37 @@
-# -*- coding: utf-8 -*-
-"""
-Created on Mon Apr  8 07:32:52 2019
-
-@author: Priyansh
-"""
-
+from tkinter import *
+from PIL import ImageTk, Image
+import os
 import tweepy,csv,re
 from textblob import TextBlob
 import matplotlib.pyplot as plt
+
 class SentimentAnalysis:
     def __init__(self):
         self.tweets = []
         self.tweetText = []
     def DownloadData(self):
-        consumerKey = 'your consumerkey'
-        consumerSecret = 'your consumersecretkey'
-        accessToken = 'your accesstokenkey'
-        accessTokenSecret = 'your accesstokensecretkey'
+        consumerKey = 'w4JcsEOK6XFg14wtJGXXVHVYD'

```

Comment:
Utkarsh You have to change these key-value pairs to any string as this will affect your security credentials